tools/src/commands/RunStories.ts

```diff
@@ -0,0 +1,241 @@
+import spawnAsync from '@expo/spawn-async';
+import { IosPlist } from '@expo/xdl';
+import fs from 'fs';
```

Comment:
FWIW, you can switch to `fs-extra` and replace all sync commands with `async` ones, but possibly this would not improve anything as you seem to process the action linearly without any synchronous waitings in loops 😉